apis/apps/v1alpha1/configconstraint_types.go

```diff
@@ -289,6 +289,14 @@ type ShellTrigger struct {
 	//
 	// +optional
 	Sync *bool `json:"sync,omitempty"`
+
+	// Specifies whether to reconfigure dynamic parameters individually or in a batch.
+	// - Set to 'True' to execute the reload action in a batch, incorporating all parameter changes.
+	// - Set to 'False' to execute the reload action for each parameter change individually.
+	// The default value is 'False'.
+	//
+	// +optional
+	BatchReload *bool `json:"batchRefresh,omitempty"`
```

Comment:
Maybe you can add a new field to define the format of batchaction instead of hard coding.

```
batchParamtersTemplate: |-
  {{- range $pKey, $pValue := $.arg0 }} 
  {{ printf "%s\t%s" $pKey $pValue }}
  {{- end }}


```

Reply:
Yes, I agree that the format I've currently defined lacks flexibility. Indeed, the format for batch input of key-value pairs can vary significantly. However, I believe this format still requires careful design because, once released, we must ensure forward compatibility. If the specific format parameters we define now are not optimal, changing them later could be problematic. Therefore, I've decided to leave this parameter undefined for the time being, establish a basic format first, and then await further feedback from users. 

The Go template solution mentioned in your comment appears somewhat complex for the average user, in my opinion. I'm exploring other options for custom formatting. Do you have any alternative suggestions?

Reply:
Fixed in da1971c221782b17a9fa4925ad56d033e04cc925